setup.py

```diff
@@ -28,7 +28,8 @@
     package_dir={"": "src"},
     install_requires=[
         "amazon-braket-sdk>=1.35.0",
-        "pennylane==0.29.1",
+        # "pennylane=0.30.0"
+        "pennylane @ git+https://github.com/PennyLaneAI/pennylane.git",

```

Comment:
Not sure whether this needs updating, or only should be updated when PL0.30 is released?

NB: When I tried out the local code on my machine, I had to deactivate the version checking in PennyLane, because `0.30.0-dev` was not considered to be `>=0.30.0` :thinking: 
```
DeviceError: The braket.local.ahs plugin requires PennyLane versions >=0.30.0, however PennyLane version 0.30.0-dev is installed.
```

Reply:
I am also not sure how to handle that up until the release this whole feature is dependent on accessing the dev version of PennyLane, but I expect it to be dealt with in the main PR, not in the documentation update. Whatever we do there, this one will just follow suit.

Reply:
Fair enough :)